.github/workflows/validate.yml

```diff
@@ -65,6 +65,10 @@ jobs:
         run: |
           go fix ./...
           git diff --exit-code
+      - name: run modernize
+        run: |
+          go run golang.org/x/tools/gopls/internal/analysis/modernize/cmd/modernize@latest -fix -test ./...
```

Comment:
We should avoid using `@latest` here. It's better to pin a stable version for reproducibility and reliability.
WDYT?

Reply:
Of course I might be wrong but only time can tell.